picard/collection.py

```diff
@@ -106,23 +106,24 @@ def request_finished(document, reply, error):
                 echo=log.error
             )
             return
-        collection_list = document.metadata[0].collection_list[0]
-        if "collection" in collection_list.children:
+        if document and "collection" in document:
```

Comment:
Is `document` ever a falsy value if `error` is not a truthy one? That would be really weird.